source/gui/build_appimage.sh

```diff
@@ -0,0 +1,97 @@
+#! /bin/sh
+
+#
+#  Copyright 2020 PasDoc developers.
+#
+#  This file is part of "PasDoc".
+#
+#  "PasDoc" is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  "PasDoc" is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with "PasDoc"; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+#
+
+# Author: Silvio Clecio (silvioprog)
+
+# TODO: add "pasdoc_gui.appdata.xml" as soon as this issue is solved: https://github.com/linuxdeploy/linuxdeploy/issues/31
+
+set -e
```

Comment:
Can you extend this, not only `set -e` but also `set -eu`?

See http://redsymbol.net/articles/unofficial-bash-strict-mode/ .

Reply:
I'm not sure if `-u` is available in all `sh` versions. I have been using [`shellcheck`](https://www.shellcheck.net) (as VSCode extension) for validade/format the script.